File: Oxygene/All/SharedUI/SharedUI.Shared/AppDelegate.Cocoa.pas
namespace SharedUI.Shared;
{$IF COCOA}

type
  [NSApplicationMain, IBObject]
  AppDelegate = public partial class(INSApplicationDelegate)
  public

    method applicationDidFinishLaunching(notification: NSNotification);
     begin
      start();
    end;

    //
    // Add Cocoa-specific code here
    //

  end;

{$ENDIF}
end.

File: Mercury/All/SharedUI/SharedUI.Shared/AppDelegate.Cocoa.vb
#if TOFFEE Then
<NSApplicationMain>
<IBObject>
Public Partial Class AppDelegate
  Inherits INSApplicationDelegate

  Public Sub applicationDidFinishLaunching(notification As NSNotification)
	start()
  End Sub

  '
  ' Add Cocoa-specific code here
  '

End Class
#End If